lib/v2/screens/profile_screens/citizenship/interactor/viewmodels/citizenship_bloc.dart

```diff
@@ -16,6 +18,8 @@ class CitizenshipBloc extends Bloc<CitizenshipEvent, CitizenshipState> {
       } else {
         yield state.copyWith(pageState: PageState.loading, profile: event.profile, score: event.score);
         var results = await GetReferredAccountsUseCase().run();
+        var results2 = await GetCitizenshipDataUseCase().run();
+        yield SetDataStateMapper().mapResultsToState(state, results2);
         yield SetValuesStateMapper().mapResultToState(state, results);
```

Comment:
The correct thing would have been to do everything in a single mapper and usecase but if it works I see no problem.

Reply:
I know but the first mapper had so much logic on it, did'nt want to add more lol